src/1.Challenges.tex

```diff
@@ -45,9 +45,9 @@ \section{Inertia}
 
 \(2.59\cdot10^{10} m/hectoclarke\)
 
-\(2.59\cdot10^{12} m/kiloclarke\)
+\(2.59\cdot10^{11} m/kiloclarke\)
 
-\(2.59\cdot10^{13} m/day\)
+\(2.59\cdot10^{12} m/day\)
```

Comment:
Should be 10^13 m/day, the same as the SI example.

Reply:
Fixed